File: COMPILADOR/kokoslan_/src/src/java/eval/KoKoAtomValue.java
/**
 @author loriacarlos@gmail.com
 
*/
package src.java.eval;
import java.io.*;

public class KoKoAtomValue<T> implements KoKoValue{
   private T value;
   public T getValue(){return this.value;}
   
   public KoKoAtomValue(T value){
      this.value = value;
   }
   public String toString(){
      return this.value.toString();
   }
}

File: COMPILADOR/kokoslan/src/KoKoKotlin/eval/KoKoAtomValue.kt
/**
 * @author loriacarlos@gmail.com
 */
package KoKoKotlin.eval

import java.io.*

open class KoKoAtomValue<T>(val value: T) : KoKoValue {
    override fun toString(): String {
        return this.value.toString()
    }
}